src/iexamine.cpp

```diff
@@ -805,6 +805,52 @@ void iexamine::cardreader_robofac( player &p, const tripoint &examp )
     }
 }
 
+void iexamine::cardreader_foodplace( player &p, const tripoint &examp )
+{
+    bool open = false;
+    if( ( p.is_wearing( itype_id( "foodperson_mask" ) ) ||
+          p.is_wearing( itype_id( "foodperson_mask_on" ) ) ) &&
+        query_yn( _( "Press mask on the reader?" ) ) ) {
+        p.mod_moves( -100 );
+        for( const tripoint &tmp : g->m.points_in_radius( examp, 3 ) ) {
+            if( g->m.ter( tmp ) == t_door_metal_locked ) {
+                g->m.ter_set( tmp, t_door_metal_c );
+                open = true;
+            }
+        }
+        if( open ) {
+            add_msg( _( "You press your face on the reader." ) );
+            add_msg( m_good, _( "The nearby doors are unlocked." ) );
+            sounds::sound( examp, 6, sounds::sound_t::speech, "\"Hello Foodperson.  Welcome home.\"", true,
```

Comment:
This is still untranslated even within #31489.

Reply:
Damn ><